libs/clients/syslumenn/src/lib/syslumennClient.service.ts

```diff
@@ -242,6 +247,18 @@ export class SyslumennService {
     return (temporaryEventLicences ?? []).map(mapTemporaryEventLicence)
   }
 
+  async sealDocuments(documents: string[]): Promise<InnsigludSkjol> {
+    const { id, api } = await this.createApi()
+    const explination = 'Rafrænt undirritað vottorð'
+    return await api.innsiglaSkjolPost({
+      skeyti: {
+        audkenni: id,
+        skyring: explination,
```

Comment:
Sorry for the nitpick :sweat_smile: 
```suggestion
    const explanation = 'Rafrænt undirritað vottorð'
    return await api.innsiglaSkjolPost({
      skeyti: {
        audkenni: id,
        skyring: explanation,
```

Reply:
Ahhh it was just a copy paste from sealDocument (singular). I'll change it in both places.